index.html

```diff
@@ -2179,7 +2256,9 @@ <h1>Directory Service API Specification</h1>
             The <a>TM</a> should describe all interaction affordances and indicate which ones
             are required (i.e. mandatory by this spec). Moreover, it should use placeholders
             for deployment-specific attributes.
+            Informative affordances (JSONPath and XPath) should be defined as such.
```

Comment:
JSONPath is definitely informative since there is no spec we can rely on.  XPath though can be normative, but optional (and SPARQL the same).

Reply:
XPath is not normative. We had a consensus on this:
- https://github.com/w3c/wot-discovery/issues/238

Reply:
ok, will let it go for now and review the record.